pkg/driver/executor/executor.go

```diff
@@ -640,9 +639,10 @@ func (ex *Executor) deleteVolume(ctx context.Context, machineName string) error
 	return nil
 }
 
-// getMachineByProviderID fetches the data for a server based on a provider-encoded ID.
-func (ex *Executor) getMachineByID(ctx context.Context, serverID string) (*servers.Server, error) {
-	klog.V(2).Infof("finding server with [ID=%q]", serverID)
+// GetMachineByID fetches the data for a server based on a provider-encoded ID.
+func (ex *Executor) GetMachineByID(ctx context.Context, providerID string) (*servers.Server, error) {
```

Comment:
Can you please change the name of this method to `GetMachineByProviderID` so that it's clear?